### PR TITLE
Bug/lga 3266 sonarcloud coverage

### PR DESCRIPTION
**Pull request by TawneeOwl**
## What does this pull request do?

Updates the sonarcloud config to include coverage of code.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,7 +27,9 @@ jobs:
     - name: Unit test with pytest
       run: |
         pip install pytest pytest-cov
-        pytest tests/unit_tests --doctest-modules --junitxml=junit/unit-test-results.xml --cov=com --cov-report=xml --cov-report=html
+        pytest tests/unit_tests --doctest-modules --junitxml=junit/unit-test-results.xml
+        coverage run -m pytest tests/unit_tests
```
Comment (BenMillar-MOJ) [inline .github/workflows/test.yml:31]: Are we running `pytest` twice?
Can this match the implementation in the Civil Case API? 
[https://github.com/ministryofjustice/laa-civil-case-api/blob/main/.github/workflows/test.yml#L25](https://github.com/ministryofjustice/laa-civil-case-api/blob/main/.github/workflows/test.yml#L25)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -37,6 +39,12 @@ jobs:
       # Use always() to always run this step to publish test results when there are test failures
       if: ${{ always() }}
```
Comment (BenMillar-MOJ) [inline .github/workflows/test.yml:40]: Can this be removed, now that we are using `Coverage.py` for the coverage?

File: sonar-project.properties [hunk 3]
```diff
@@ -3,4 +3,5 @@ sonar.projectKey=ministryofjustice_laa-access-civil-legal-aid
 
 # relative paths to source directories. More details and properties are described
 # in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
-sonar.sources=src
+sonar.sources=.
+sonar.tests=tests/unit_tests
```
Comment (BenMillar-MOJ) [inline sonar-project.properties:7]: Should Sonar Cloud only be looking at the unit tests or the broader test directory?
[https://github.com/ministryofjustice/laa-civil-case-api/blob/main/sonar-project.properties#L7](https://github.com/ministryofjustice/laa-civil-case-api/blob/main/sonar-project.properties#L7)

**Review by BenMillar-MOJ — CHANGES_REQUESTED** [review]

File: sonar-project.properties (hunk 3 above)
Comment (TawneeOwl) [reply sonar-project.properties:7]: The sonar source needs to look at our whole project and the coverage is just code coverage of unit tests, our functional tests are looking at the service.

File: .github/workflows/test.yml (hunk 2 above)
Comment (TawneeOwl) [reply .github/workflows/test.yml:40]: We still need the junit test results to show errors with the unit tests beside coverage.

**Review by BenMillar-MOJ — APPROVED** [review]
LGTM, thanks for this! 👍

**Review by digitalali-moj — APPROVED** [review]
Nice work, thanks :)

**Pull request merged**